Can't use unless babel-runtime installed

## Expected behaviour

Babel-runtime to be installed with the other packages when I install twtich-js

## Actual behaviour

Have to manually install babel-runtime.

## Steps to reproduce

Tell us how to reproduce this issue (please provide a code sample and/or working demo)

## Your environment

* Operating system(s):
* NodeJS version: 8.1.0


I was having issues getting this going today after using tmi.js forever, I finally realized it is because I didn't have babel-runtime installed.  If it can't be installed when I install twitch-js, can the docs be updated to reflect that it needs to be installed first.

using your exact code from what you linked this is what I get.

C:\NodeJS>node app.js
module.js:471
    throw err;
    ^

Error: Cannot find module 'babel-runtime/regenerator'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\NodeJS\node_modules\twitch-js\lib\api.js:3:20)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)

--- a/.babelrc
+++ b/.babelrc
@@ -1,10 +1,10 @@
 {
+  "presets": ["env", "stage-3"],
   "plugins": [
     "lodash",
     ["transform-runtime", {
       "polyfill": false,
       "regenerator": true
     }]
-  ],
-  "presets": ["env", "stage-0"]
+  ]
 }

--- a/.gitignore
+++ b/.gitignore
@@ -40,6 +40,9 @@ build/Release
 # Deployed apps should consider commenting this line out:
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules
+package-lock.json
+
+
 # =========================
 # Operating System Files
 # =========================

--- a/package.json
+++ b/package.json
@@ -57,6 +57,7 @@
     "build": "run-p build:**"
   },
   "dependencies": {
+    "babel-runtime": "^6.26.0",
     "lodash": "^4.17.4",
     "node-fetch": "^1.7.3",
     "request": "^2.83.0",
@@ -74,7 +75,7 @@
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.6.1",
-    "babel-preset-stage-0": "^6.24.1",
+    "babel-preset-stage-3": "^6.24.1",
     "babelify": "^8.0.0",
     "browserify": "^14.5.0",
     "compression-webpack-plugin": "^1.1.3",

